### ISSUE TITLE
Hackers

### ISSUE THREAD

**Issue by Faris90** [Picoseconds/sanctuary#28]
Will hackers be able to ruin this type of server?

**Comment by Erisfiregamer1** [comment]
I swear to the fucking gods Faris90, I remember you. Remember woomy arras? We never forgot, and leave this fucking place.

**Comment by Picoseconds** [comment]
Perhaps. I suggest you try.

**Comment by bqini** [comment]
answer is yes

**Comment by Justgamer101** [comment]
yes, the packets should have been changed to make hacking more difficult

**Comment by Faris90** [comment]
@Erisfiregamer1
"**i SWeaR tO The FuCKING goDS FAriS90, I ReMEMBER yOU. ReMemBeR WOoMY ArRaS? we neVEr FOrgoT, and LEaVe tHis FucKiNG PLaCe.**"
Move on, I'm done with this arras drama shit you loser.

**Comment by bqini** [comment]
lol

**Comment by eris-webserv** [comment]
> @Erisfiregamer1 "**i SWeaR tO The FuCKING goDS FAriS90, I ReMEMBER yOU. ReMemBeR WOoMY ArRaS? we neVEr FOrgoT, and LEaVe tHis FucKiNG PLaCe.**" Move on, I'm done with this arras drama shit you loser.

I, um, said that a LONG while ago, and ok then im sorry. Also, why did you post on my repo too?
